definitions/npm/apollo-link-http_v1.2.x/flow_v0.28.x-/apollo-link-http_v1.2.x.js

```diff
@@ -0,0 +1,20 @@
+declare module "apollo-link-http" {
+  declare export interface FetchOptions {
+    uri?: string;
+    //todo change to type of Global['fetch']
+    fetch?: any;
+    includeExtensions?: boolean;
+    credentials?: string;
+    headers?: any;
+    fetchOptions?: any;
```

Comment:
Should I use `mixed` here instead (for both `headers` and `fetchOptions`)?

Reply:
`any` is fine, as it fetches the `ts` definition.